scripts/tests/lib.sh

```diff
@@ -20,11 +20,46 @@ function relay2() {
 
 function install_iptables_drop_rules() {
```

Comment:
Could we document what the parameters do? Are they the stacks that allow traffic?

Reply:
Renamed to `force_relayed_connections` and added comments.